res/games/gameitem.jinja

```diff
@@ -0,0 +1,120 @@
+{%- macro playermacro(player, me, iconpath, teamalign) %}
+        {% set player_login = player.login %}
+        {% set player_country = "" %}
+        {% set player_global_rating = "???" %}
+        {% if player.login is defined %}
+            {% set player_country = iconpath~player.country~".png" %}
+            {% set player_global_rating = player.rating_estimate %}
+        {% endif %}
+
+        {% set width = 0 %}
+            {% if "-" in player.login %}
```

Comment:
Did you try replacing the '-' with a non-breaking hyphen `&#8209;`? Jinja should allow for replacing substrings. Also whatever the solution, we should add a (jinja) comment explaining why it's needed.